bundles/org.openhab.binding.huesync/README.md

```diff
@@ -0,0 +1,288 @@
+# HueSync Binding
+<!-- markdownlint-disable MD033 -->
+
```

Comment:
Prevents a markdown warning: Missing an empty line after the Markdown header (#).
```suggestion

<!-- markdownlint-disable MD033 -->
```

Reply:
➡️ a7b461244696b450379dfe3b49bd3611038e6082